packages/ui/src/components/skeleton-loader/skeleton-loader.native.tsx

```diff
@@ -0,0 +1,94 @@
+import { useLayoutEffect } from 'react';
+import { StyleSheet } from 'react-native';
+import Animated, {
+  Easing,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+
+import { useTheme } from '@shopify/restyle';
+import { LinearGradient } from 'expo-linear-gradient';
+
+import { Box } from '../box/box.native';
+
+const AnimatedLinearGradient = Animated.createAnimatedComponent(LinearGradient);
+
+export const ANIMATION_DURATION = 1_000;
+const ANIMATION_EASING = Easing.ease;
+const LEFT_POS_START = -100;
+const LEFT_POS_END = 100;
```

Comment:
Is there a reason these are all caps? Typically in the extension we only use this when constants are shared globally. Should these be?

Reply:
**NO REASON** 😄 

I can update, thanks

Reply:
We have a handful of conventions mentioned in Notion https://www.notion.so/trustmachines/Naming-constants-1a66b44cd8234f11aadaadff5e45a3f1?pvs=4

Would love to grow this and add more

Reply:
@kyranjamie, can these conventions and standards be made public? I need to request access to view them.

Reply:
@adrianocola notion is not public but here's what it says: 
![Screenshot 2024-09-05 at 06 08 45](https://github.com/user-attachments/assets/89396be9-73fc-4709-a937-19e855fa9e6e)

I will rename them here so we can merge more efficiently